### PR TITLE
Added Possibility for ESpeak Config

### PR DESCRIPTION
**Pull request by Genei180**
## Description
Fixes my Issue Described in https://github.com/MycroftAI/mycroft-core/issues/3018

The Following Options are available to Set via the Mycroft Config:
- Amplitude
- Gap
- Capital
- Pitch
- Speed
(for more Information see `espeak --help`

Example Config:
```
{  
  "lang": "de-de",  
  "tts": {  
    "module": "espeak",  
    "espeak": {  
      "lang": "german-mbrola-5",  
      "voice": "german-mbrola-5",  
      "speed": "135",  
      "amplitude": "80",  
      "pitch": "20"  
    }  
  },  
  "max_allowed_core_version": 21.2  
}  
```

## How to test
Install Espeak with the Mbrola Voice after the Mycroft Documentation for Installing German:
https://mycroft-ai.gitbook.io/docs/using-mycroft-ai/customizations/languages/german

## Contributor license agreement signed?
CLA [x] (Whether you have signed a [CLA - Contributor Licensing Agreement](https://mycroft.ai/cla/)

## Note
Please be extra Careful with my Git Commit History as I tried cleaning it up and I am not sure if I succeeded.

### CODE REVIEW

**Comment by Genei180** [comment]
By the way could I please count this Towards my Hacktoberfest Entries?
(If it passes)
Thank you :)

File: mycroft/tts/espeak_tts.py [hunk 1]
```diff
@@ -32,8 +32,32 @@ def get_tts(self, sentence, wav_file):
         Returns:
             tuple ((str) file location, None)
         """
-        subprocess.call(['espeak', '-v', self.lang + '+' + self.voice,
-                         '-w', wav_file, sentence])
+
+        # Create Argument String for Espeak
+        arguments = ['espeak', '-v', self.lang + '+' + self.voice]
+        argument = self.config.get("amplitude")
+        if argument:
```
Comment (krisgesling) [inline mycroft/tts/espeak_tts.py:39]: I know it's not a complex function, however I think we can make it clearer by using the name of each argument as the variable names eg:
```
amplitude = self.config.get...
if amplitude:
  ...
gap = ...
```

File: mycroft/tts/espeak_tts.py [hunk 2]
```diff
@@ -32,8 +32,32 @@ def get_tts(self, sentence, wav_file):
...
+            arguments.append("-a "+argument)
+
+        argument = self.config.get("gap")
```
Comment (krisgesling) [inline mycroft/tts/espeak_tts.py:42]: The rest of this module is using single quotes - would be great to keep that consistency.

**Review by krisgesling — COMMENTED** [review]
Hey there,

Thanks for contributing to Mycroft! :tada:

This looks like a great addition to espeak. We will be pulling this out to be a plugin soon but given the change is relatively minor we'll merge it in and then pull it all out together.

A couple of minor points below - if you can fix them up it should be ready to merge.

Thanks again :)

File: mycroft/tts/espeak_tts.py (hunk 1 above)
Comment (Genei180) [reply mycroft/tts/espeak_tts.py]: Thanks for the Feedback :)
Improved it

File: mycroft/tts/espeak_tts.py (hunk 2 above)
Comment (Genei180) [reply mycroft/tts/espeak_tts.py]: Also thanks for the Feedback :)
Updated it.

**Comment by Genei180** [comment]
About the Plugin, 
I was kind of Surprised that I could actually find the Code in Mycroft core :)

If I can help with the Plugin, let me know (:
But please be aware that it may take some time...

Thanks for your kind feedback!

**Review by krisgesling — APPROVED** [review]
Thanks for fixing those up - it all seems to be working great.

**Comment by krisgesling** [comment]
I've also added these options to the documentation:
https://mycroft-ai.gitbook.io/docs/using-mycroft-ai/customizations/tts-engine#espeak

Thanks for such a clear PR description :)

**Pull request merged**